man/avocado.rst

```diff
@@ -28,62 +28,106 @@ The following list of options are builtin, application level `avocado`
 options. Most other options are implemented via plugins and will depend
 on them being loaded::
 
- -h, --help             show this help message and exit
- -v, --version          show program's version number and exit
- --config CONFIG_FILE   Use custom configuration from a file
+  -h, --help            show this help message and exit
+  -v, --version         show program's version number and exit
+  --config [CONFIG_FILE]
+                        Use custom configuration from a file
+  --show [STREAM[:LVL]]
+                        List of comma separated builtin logs, or logging
+                        streams optionally followed by LEVEL (DEBUG,INFO,...).
+                        Builtin streams are: "test": test output; "debug":
+                        tracebacks and other debugging info; "app":
+                        application output; "early": early logging of other
+                        streams, including test (very verbose); "remote":
+                        fabric/paramiko debug; "all": all builtin streams;
+                        "none": disables regular output (leaving only errors
+                        enabled). By default: 'app'
+  -s, --silent          disables regular output (leaving only errors enabled)
 
 Real use of avocado depends on running avocado subcommands. This a typical list
 of avocado subcommands::
 
- run         Run one or more tests (native test, test alias, binary or script)
- list        List available test modules
- sysinfo     Collect system information
- multiplex   Generate a list of dictionaries with params from multiplex file(s)
- plugins     List all plugins loaded
- distro      Shows detected Linux distribution
- datadir     List all relevant directories used by avocado
+    config              Shows avocado config keys
+    diff                Shows the difference between 2 jobs.
+    distro              Shows detected Linux distribution
+    exec-path           Returns path to avocado bash libraries and exits.
+    list                List available tests
+    multiplex           Tool to analyze and visualize test variants and params
+    plugins             Displays plugin information
+    run                 Runs one or more tests (native test, test alias,
+                        binary or script)
+    sysinfo             Collect system information
 
 To get usage instructions for a given subcommand, run it with `--help`. Example::
 
  $ avocado multiplex --help
- usage: avocado multiplex [-h] [--filter-only [FILTER_ONLY [FILTER_ONLY ...]]]
-                          [--filter-out [FILTER_OUT [FILTER_OUT ...]]] [-t]
-                          [-c]
-                          multiplex_files [multiplex_files ...]
-
- positional arguments:
-   multiplex_files       Path(s) to a multiplex file(s)
+ usage: avocado multiplex [-h] [--system-wide] [-c] [-d] [-t] [-i]
```

Comment:
Actually I believe there should be another category for multiplex. Currently it's part of `OPTIONS` as the only command. The rest of the commands have their own categories...